app/lib/frontend/templates/views/layout.mustache

```diff
@@ -15,8 +15,8 @@
   <meta name="twitter:card" content="summary" />
   <meta name="twitter:site" content="@dart_lang" />
   <meta property="og:site_name" content="Dart Packages" />
-  <title>{{title}}</title>
-  <meta property="og:title" content="{{title}}" />
+  <title>{{& title}}</title>
+  <meta property="og:title" content="{{& title}}" />
```

Comment:
Is there any harm in escaping this twice? Or is there any problem using HTML escape provided by mustache instead of he custom one we have?

I suppose it's because it's a page title?

Reply:
`{{&` is the non-escaped content. We already escaped the `title` variable in the preparation step.